pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -524,6 +526,58 @@ def _check_width(self, width, pbc):
                 return True
         return False
 
+    def get_spherical_harmonics(self, l, m, cutoff_radius=np.inf, rotation=None):
+        """
```

Comment:
Would be cleaner if it mentions that it operates on `self.vecs`.  This makes sense, but less mental burden on users is always better.

Reply:
> This makes sense, but less mental burden on users is always better.
💯